helm/acs-sso-example/values.yaml

```diff
@@ -73,8 +71,8 @@ keycloakx:
             webOrigins: >-
               {{ index (include "alfresco-common.known.urls" $ | mustFromJson) "known_urls" }}
         users:
-          -  # -- default Alfresco admin user
-            username: admin
+          # -- default Alfresco admin user
+          - username: admin
```

Comment:
syntax may look odd but I think it was intended to have the doc entry only for the username field, now it is attached to the entire object